src/seedu/addressbook/ui/TextUi.java

```diff
@@ -75,45 +64,54 @@ private boolean isCommentLine(String rawInputLine) {
      * @return command (full line) entered by the user
      */
     public String getUserCommand() {
-        out.print(LINE_PREFIX + "Enter command: ");
+        out.print(outFormatter.formatLine("Enter command: "));
         String fullInputLine = in.nextLine();
 
         // silently consume all ignored lines
         while (shouldIgnore(fullInputLine)) {
             fullInputLine = in.nextLine();
         }
 
-        showToUser("[Command entered:" + fullInputLine + "]");
+        showToUserFormatted("[Command entered:" + fullInputLine + "]");
         return fullInputLine;
     }
 
-
     public void showWelcomeMessage(String version, String storageFilePath) {
         String storageFileInfo = String.format(MESSAGE_USING_STORAGE_FILE, storageFilePath);
-        showToUser(
-                DIVIDER,
-                DIVIDER,
-                MESSAGE_WELCOME,
-                version,
-                MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE,
-                storageFileInfo,
-                DIVIDER);
+        showToUser(outFormatter.formatLinesWithDividers(2, 1, MESSAGE_WELCOME, version,
+                MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE, storageFileInfo));
     }
 
-    public void showGoodbyeMessage() {
-        showToUser(MESSAGE_GOODBYE, DIVIDER, DIVIDER);
+    public static void main(String[] args) {
+        new TextUi().showToUser("hello", "abs");
```

Comment:
this code is not useful! i guess you wrote it to test the behavior? please remove it ;-)